ioctl/client_test.go

```diff
@@ -0,0 +1,257 @@
+	config.ReadConfig, err = getTempConfig(t, &cfg)
+	c = NewClient()
+	out, err = c.GetAddress(testDatas[2].in)
+	r.Error(err)
+	r.Contains(err.Error(), `use "ioctl config set defaultacc ADDRESS|ALIAS" to config default account first`)
+	r.Equal("", out)
+
+	cfg = config.Config{
+		Aliases: map[string]string{
+			testDatas[0].in: testDatas[0].out,
+			"bbb":           "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hbbb",
+		},
+		DefaultAccount: config.Context{AddressOrAlias: ""},
+	}
+	config.ReadConfig, err = getTempConfig(t, &cfg)
+	c = NewClient()
+	out, err = c.GetAddress(testDatas[0].in)
+	r.NoError(err)
+	r.Equal(testDatas[0].out, out)
+
+	cfg = config.Config{
+		Aliases: map[string]string{
+			"ccc":           "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hccc",
+			testDatas[3].in: testDatas[3].out,
+			"bbb":           "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hbbb",
+		},
+		DefaultAccount: config.Context{AddressOrAlias: testDatas[3].in},
+	}
+	config.ReadConfig, err = getTempConfig(t, &cfg)
+	c = NewClient()
+	out, err = c.GetAddress(testDatas[3].in)
+	r.NoError(err)
+	r.Equal(testDatas[3].out, out)
+}
+
+func TestNewKeyStore(t *testing.T) {
+	testWallet := filepath.Join(os.TempDir(), "ksTest")
+	defer testutil.CleanupPath(t, testWallet)
+	config.ReadConfig.Wallet = testWallet
+	c := NewClient()
+	defer c.Stop(context.Background())
+	keyStore := c.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
+	require.NotNil(t, keyStore)
+	keyStore = c.NewKeyStore(config.ReadConfig.Wallet, keystore.LightScryptN, keystore.LightScryptP)
+	require.NotNil(t, keyStore)
+}
+
+func TestGetAliasMap(t *testing.T) {
+	cfg := config.Config{
+		Aliases: map[string]string{
+			"aaa": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95haaa",
+			"bbb": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hbbb",
+			"ccc": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hccc",
+		},
+	}
+	require.NoError(t, writeTempConfig(t, &cfg))
+	cfg, err := config.LoadConfig()
+	defer testutil.CleanupPath(t, config.ConfigDir)
+	require.NoError(t, err)
+	config.ReadConfig = cfg
+
+	exprAliases := map[string]string{
+		cfg.Aliases["aaa"]: "aaa",
+		cfg.Aliases["bbb"]: "bbb",
+		cfg.Aliases["ccc"]: "ccc",
+	}
+	c := NewClient()
+	defer c.Stop(context.Background())
+	result := c.GetAliasMap()
+	require.Equal(t, exprAliases, result)
+}
+
+func TestWriteConfig(t *testing.T) {
+	config.ConfigDir = os.Getenv("HOME") + "/.config/ioctl/default"
+	if !fileutil.FileExists(config.ConfigDir) {
+		err := os.MkdirAll(config.ConfigDir, 0700)
+		require.NoError(t, err)
+	}
+	config.DefaultConfigFile = config.ConfigDir + "/config.default"
+
+	cfg := config.Config{
+		Aliases: map[string]string{
+			"aaa": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95haaa",
+			"bbb": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hbbb",
+			"ccc": "io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95hccc",
+		},
+		DefaultAccount: config.Context{AddressOrAlias: "ddd"},
+	}
+	c := NewClient()
+	defer c.Stop(context.Background())
+	err := c.WriteConfig(cfg)
```

Comment:
after write config, verify c.Alias is correct? like
`r.Equal("aaa", c.Alias(io1cjh35tq9k8fu0gqcsat4px7yr8trh75c95haaa))` ?